.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ commands:
             }
 
             echo 'Checking changes'
-            changes=`git diff --name-only origin/master...$CIRCLE_BRANCH | { grep -Ev '^packages/|yarn.lock|bear.png|.editorconfig' || true; }`
+            changes=`git diff --name-only origin/master...$CIRCLE_BRANCH | { grep -Ev '^packages\/(?!storybook)|yarn.lock|bear.png|.editorconfig' || true; }`
```

Comment:
Can you add a changelog entry to the root, just explaining the change (the package number is incidental, it's more to track changes into the repo)